main.cpp

```diff
@@ -313,171 +345,153 @@ struct Staff
+    {
+    }
+void Entertainment::workForLongHaul()
+    {
+    }
 
 struct Facilities
 {
-    // size of swimming pool in hectars
     float sizeOfPoolInHectars = 200;
-    // number of shops opened 24/7
     int numOfShopsOpened24_7 = 20;
-    // opening hours of spa
     std::string openingHoursOfSpa = "10am - 10pm";
-    // number of gym equipment
     int numOfGymEquipment = 40;
-    // number of restauruants available
     int numOfRestauruantsAvailable = 25;
 
-    // keep passengers busy
     void keepPassengersBusy(int activitiesPlanned);
-    // entertain children
     void entertainChildren(int dayCareServiceSlots);
-    // allow passengers to relax
     void allowPassengersToRelax(int numOfRelaxingSongInPlaylist);
 };
 
-/*
-Thing 9) SafetyProcedure
-5 properties:
-    1) number of floors (int)
-    2) number of personnel on board (int)
-    3) number of lifeboats (int)
-    4) helpline number (int)
-    5) meeting point (char)
-3 things it can do:
-    1) inform crew and passengers safety information
-    2) display floorplan
-    3) display exit route
- */
+void Facilities::keepPassengersBusy(int activitiesPlanned)
+    {
+        activitiesPlanned = 20;
+    }
+void Facilities::entertainChildren(int dayCareServiceSlots)
+    {
+        dayCareServiceSlots = 5;
+    }
+void Facilities::allowPassengersToRelax(int numOfRelaxingSongInPlaylist)
+    {
+        numOfRelaxingSongInPlaylist = 200;
+    }
 
 struct SafetyProcedure
 {
-    // number of floors
     int numOfFloors = 6;
-    // number of personnel on board
     int numOfPersonnelOnBoard = 1500;
-    // number of lifeboats
     int numOfLifeboats = 80;
-    // helpline number (int)
     int helplineNum = 10291092;
-    // meeting point
     char meetingPoint = 'C';
 
-    // inform crew and passengers safety information
-    bool informSafetyInformation(bool SentViaEmail, bool broadcastedOnPA); //returns if information has been shared.
-    // display floorplan
-    bool displayFloorplan(); //returns if floorplan has been displayed.
-    // display exit route
-    bool displayExitRoute(); //returns if exit route has been displayed.
+    auto informSafetyInformation(bool broadcastedOnPA);
+    void displayFloorplan();
+    void displayExitRoute();
 };
 
-/*
-Thing 10) Cruise Ship
-5 properties:
-    1) Staff
-    2) BuildSpecifications
-    3) Entertainment
-    4) Facilities
-    5) SafetyProcedure
-3 things it can do:
-    1) travel for 10 days without stopovers
-    2) ferry 5000 passengers
-    3) sink
- */
+auto::SafetyProcedure::informSafetyInformation(bool broadcastedOnPA)
+    {
+        if (broadcastedOnPA == true)
+        {
+            std::string done = "It is safe";
+            return done;
+        }
+        else
+        {
+            std::string notDone = "It is NOT safe";
+            return notDone;
+        }
+    }
+void::SafetyProcedure::displayFloorplan()
+    {
+    }
+void::SafetyProcedure::displayExitRoute()
+    { 
+    }
```

Comment:
Fix this indentation
Dont' write `returnType::ClassName::functionName`
the correct syntax involves using `space` between the return type and the function name
```
void SafetyProcedure::displayFloorplan()
```
- [ ] revise